src/main/java/frc/robot/util/FieldPose2024.java

```diff
@@ -47,4 +47,8 @@ public FieldPose2024(Pose2d bluePose) {
         super(FieldWidthMeters, DefaultAlliance, null, bluePose);
     }
 
-}
\ No newline at end of file
+    public double DistanceTo(Pose2d location) {
```

Comment:
Minor comment: Can we add either a unit to the function name or a javadoc comment? (or both)